DPRHTML/search-results.html

```diff
@@ -66,7 +66,7 @@
       </div>
     </div>
 
-    <div class="row">
+    <div class="row"><div id="paliTextContent">
```

Comment:
Format please.

Also please ensure that 'para' parameter works. It has a dependency on paliTextContext
Refer my previous commit.